pkg/eraser/eraser.go

```diff
@@ -151,6 +162,45 @@ func removeImages(c Client, targetImages []string) error {
 	return nil
 }
 
+func isExcluded(img string, idToTagListMap map[string][]string) bool {
+	// check if img excluded by digest
+	if _, contains := excluded[img]; contains {
+		return true
+	}
+
+	// check if img excluded by name
+	if len(idToTagListMap[img]) > 0 {
+		if _, contains := excluded[idToTagListMap[img][0]]; contains {
+			return true
+		}
+	}
+
+	r := regexp.MustCompile(`(\w)+\/\*\z`)
```

Comment:
`\w` won't match dashes or periods, which are both valid in image names. The image distribution spec's [full regex](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#pulling-manifests) for image names is:
`[a-z0-9]+([._-][a-z0-9]+)*(/[a-z0-9]+([._-][a-z0-9]+)*)*`.

So to make this work, it would be
```
r := regexp.MustCompile(`[a-z0-9]+([._-][a-z0-9]+)*\/\*\z`)
```